bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/contract/form/helper/ObsUtil.java

```diff
@@ -0,0 +1,17 @@
+package org.bahmni.module.bahmnicore.contract.form.helper;
```

Comment:
Again, should not be under "contract", should be just under "org.bahmni.module.bahmnicore.util". If you want this util to be specific to form2, then just renaming the class (e.g. Form2ObsUtil" should be enough.